docs/gettingstarted/query_test_report.rst

```diff
@@ -20,6 +20,11 @@ with default format fields. To see a list of all format fields, click :ref:`here
     .. command-output:: buildtest report
        :ellipsis: 20
 
+To get the row count of the tables run ``buildtest report --row-count``. 
```

Comment:
i think you may want to rephrase this sentence and may seem a bit too direct. Perhaps you can say something along the lines
___

The `buildtest report` command will fetch all records and display them in a table format, if you want to know how many records are displayed you can use ``buildtest report --row-count`` which will display a raw count of records displayed in table. In the example below we will run ``buildtest report --row-count`` which will show number of tests in the report

___

You can then say afterward something like

"You may find it useful to fetch all failed records and determine pass/fail criteria as part of your shell script, this can be done by running ``buildtest report --row-count --fail``"